test/jdk/javax/management/monitor/ThreadPoolAccTest.java

```diff
@@ -67,7 +66,7 @@ public String getString() {
             return "";
         }
         private void setPrincipal() {
-            Subject subject = Subject.getSubject(AccessController.getContext());
+            Subject subject = Subject.current();
```

Comment:
Why change this to `Subject.current()` if you are requiring the SM to be allowed?